skrub/tests/test_fuzzy_join.py

```diff
@@ -329,37 +366,39 @@ def test_numerical_column() -> None:
     assert fj_num3.shape == (2, n_cols)
 
 
-def test_datetime_column():
+@pytest.mark.parametrize("px, assert_frame_equal_", ASSERT_TUPLES)
+def test_datetime_column(px, assert_frame_equal_):
     """
     Testing that fuzzy_join works with datetime columns.
     """
-
-    left = pd.DataFrame(
+    if px is pl:
+        pytest.xfail(reason="Module 'polars' has no attribute 'to_datetime'")
+    left = px.DataFrame(
```

Comment:
So always use 

```
df = pd.DataFrame({"A": [0, 1], "B": [0, 1]})
df = px.DataFrame(df)
```

instead of ```df = px.DataFrame({"A": [0, 1], "B": [0, 1]})``` ?

I guess for this example it's the same, but the first formulation generalizes to nulls, missing values filling, and other specificities.

Reply:
Similar example here:

```
# Numpy array with two columns
X = np.array([["alice", "charlie"], ["bob", "delta"]])
enc = GapEncoder(n_components=2, random_state=42)
X_enc_array = enc.fit_transform(X)
# Dataframe with two columns
df = px.DataFrame(X)
```

where `df = px.DataFrame(X)` could be `df = pd.DataFrame(X); df = px.DataFrame(df)`

Reply:
Let's be flexible. If the dataframe is as trivial as `df = px.DataFrame({"A": [0, 1], "B": [0, 1]})`, using `px` directly is fine.